lib/foreman_ansible/register.rb

```diff
@@ -160,14 +160,19 @@
                { :'api/v2/ansible_inventories' => [:schedule] }
     permission :import_ansible_playbooks,
                { :'api/v2/ansible_playbooks' => [:sync, :fetch] }
+    permission :view_lookup_values, {}, :resource_type => 'LookupValue'
+    permission :create_lookup_values, {}, :resource_type => 'LookupValue'
+    permission :edit_lookup_values, {}, :resource_type => 'LookupValue'
+    permission :destroy_lookup_values, {}, :resource_type => 'LookupValue'
```

Comment:
Why is this needed? I mean, I've removed those lines and it still works...

Reply:
Did you first remove these permissions from the DB? 
From the rails console: `Permission.where("name LIKE '%_lookup_values%'").destroy_all` 
And then re-run the rails server to test it.

Reply:
I've done this and it looks... not quite good.

I've destroyed permissions, but we still need to create them, to use here. Thus, I needed to run `db:seed` to re-create them (we have defined them in core), but due to Ansible plugin needing these perms to create ARM role, it failed. Thus, after I removed Ansible plugin from loading, I've re-created the permissions, then enabled the plugin and run the server. I was able to change variable values even without this block.

I mean, we already create these perms in the core, why would we try to re-create them here? Seems weird. I've checked what we do with e.g. `create_job_invocations`, since this permission is used for ARM role as well: it seems REX plugin creates it and Ansible plugin then simply uses it to define a role. As you can see, Ansible plugin doesn't call `permission :create_job_invocations`, which is right.

Since I'm not so well familiar with the internals of permission logic, @adamruzicka, maybe you can help us out?

Reply:
I forgot I need to make a change in the `db/seeds.d/020-permissions_list.rb` file for the `db:seed` command to include that file. That's why it didn't work as expected for me.
Therefore, as long as we run the `db:seed` command, there's no requirement to additionally include these permissions in the `register.rb` file, right?

Reply:
:shrug: that's what I hoped for...

Reply:
But the test failures now show what I was facing as well :/ Not sure why it complains about perms that are being created in DB by Foreman itself, isn't here some issues with load ordering?